app/src/main/java/ch/epfl/favo/util/FakeFavorList.java

```diff
@@ -7,37 +7,41 @@
 import ch.epfl.favo.favor.Favor;
```

Comment:
what is the  purpose of this file, where is it being used? and if it is  only being used for tests i think its better to move this into the test directory

Reply:
Another closed PR had this file, we should ask the author of this file for the purpose. I'll keep it there because I don't want to break things.